lib/OrdinaryDiffEqRosenbrock/src/rosenbrock_caches.jl

```diff
@@ -218,7 +218,7 @@ function alg_cache(alg::ROS3P, u, rate_prototype, ::Type{uEltypeNoUnits},
         assumptions = LinearSolve.OperatorAssumptions(true))
     grad_config = build_grad_config(alg, f, tf, du1, t)
     jac_config = build_jac_config(alg, f, uf, du1, uprev, u, tmp, du2)
-    Rosenbrock33Cache(u, uprev, du, du1, du2, k1, k2, k3, k4,
+    RosenbrockCache(u, uprev, du, du1, du2, k1, k2, k3, k4,
```

Comment:
you need to combine the ks and dus

Reply:
Ok I will store all the ks and dus in an array an make a loop or something like that